simple-web-server/src/main.rs

```diff
@@ -1,26 +1,32 @@
 use std::net::TcpListener;
+use std::net::TcpStream;
+use std::io::prelude::*;
 use std::io::{self};
+use std::borrow::Cow;
 
 fn main() {
 
-    let port= 80;
+    let port = 8080;
     let port_str = port.to_string();
     if !check_port_availability(port) {
         let stdout = io::stdout();
         let _handle = stdout.lock();
 
         // @todo - Panic or just print and exit with err 1? Sample code below
-        panic!("Unable to bind to port 80!");
+        panic!("Unable to bind to port {}!", port);
 
 //        println!("Unable to bind to port 80!");
 //        process.exit(0x0100);
     }
 
     let listener = TcpListener::bind(format!("127.0.0.1:{}", port_str)).unwrap();
-    println!("Listener started on port 80, accepting connections...");
+    println!("Listener started on port {}, accepting connections...", port);
 
-    for _stream in listener.incoming() {
```

Comment:
Here is an explanation for underscored variable names: https://medium.com/codechain/rust-underscore-does-not-bind-fec6a18115a8